__STATIC_CONTENT_MANIFEST error when bundling with Cloudflare's wrangler.

When using Hono@3.2.1 for Cloudflare's Workers, when bundling with Wrangler or ESbuild I get this error when importing `serveStatic` from `hono/cloudflare-workers`
```
✘ [ERROR] Could not resolve "__STATIC_CONTENT_MANIFEST"

    node_modules/hono/dist/adapter/cloudflare-workers/server-static-module.js:2:21:
      2 │ import manifest from "__STATIC_CONTENT_MANIFEST";
        ╵                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~

  You can mark the path "__STATIC_CONTENT_MANIFEST" as external to exclude it
  from the bundle, which will remove this error.

1 error
```
Even when I mark it as `external` and `exclude it form the bundle` When running the worker I get an error stating `Uncaught SyntaxError: Cannot use import statement outside a module`

> Even when I mark it as `external` and `exclude it form the bundle` When running the worker I get an error stating `Uncaught SyntaxError: Cannot use import statement outside a module`

Thank you @yusukebe for responding!
As noted above
> Even when I mark it as external and exclude it form the bundle When running the worker I get an error stating Uncaught SyntaxError: Cannot use import statement outside a module

Here's the command logs I get:

`With no --external flag` `error`
```
 ~/TCG_SITES/simple-renderer-base   main ±  npx esbuild app.js --bundle --outfile=dist/worker.js --loader:.html=text --loader:.md=text --loader:.csv=text --loader:.css=text --platform=node --watch --format=esm
✘ [ERROR] Could not resolve "__STATIC_CONTENT_MANIFEST"

    node_modules/hono/dist/adapter/cloudflare-workers/server-static-module.js:2:21:
      2 │ import manifest from "__STATIC_CONTENT_MANIFEST";
        ╵                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~

  You can mark the path "__STATIC_CONTENT_MANIFEST" as external to exclude it from the bundle, which
  will remove this error.

1 error
[watch] build finished, watching for changes...
^C
```

`With --external flag` no error
```
 ✘  ~/TCG_SITES/simple-renderer-base   main ±  npx esbuild app.js --bundle --outfile=dist/worker.js --loader:.html=text --loader:.md=text --loader:.csv=text --loader:.css=text --platform=node --watch --format=esm --external:__STATIC_CONTENT_MANIFEST
[watch] build finished, watching for changes...
```
`Wrangler serving compiled worker file` `error`
```
 ~/TCG_SITES/simple-renderer-base   main ±  npx wrangler dev dist/worker.js --no-bundle=true
 ⛅️ wrangler 3.0.1
------------------
wrangler dev now uses local mode by default, powered by 🔥 Miniflare and 👷 workerd.
To run an edge preview session for your Worker, use wrangler dev --remote
▲ [WARNING] Enabling Node.js compatibility mode for built-ins and globals. This is experimental and has serious tradeoffs. Please see https://github.com/ionic-team/rollup-plugin-node-polyfills/ for more details.


Your worker has access to the following bindings:
- Vars:
  - API_BASE_URL: "https://static-node-api-bjoyt36yaq-uc..."
  - SITE: "base"
⎔ Starting local server...
service core:user:renderer-base: Uncaught Error: Dynamic require of "__STATIC_CONTENT_MANIFEST" is not supported
  at core:user:renderer-base:13:11
  at core:user:renderer-base:271:48
  at core:user:renderer-base:8380:3
✘ [ERROR] MiniflareCoreError [ERR_RUNTIME_FAILURE]: The Workers runtime failed to start. There is likely additional logging output above.
```

Please let me know if this helps narrow the troubleshooting.
Also why is it necessary to always import a static package of `__STATIC_CONTENT_MANIFEST` ?

--- a/src/adapter/cloudflare-workers/serve-static-module.ts
+++ b/src/adapter/cloudflare-workers/serve-static-module.ts
@@ -1,16 +1,10 @@
 // @denoify-ignore
-// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-// @ts-ignore
 // For ES module mode
-import manifest from '__STATIC_CONTENT_MANIFEST'
 import type { Env } from '../../types'
 import type { ServeStaticOptions } from './serve-static'
 import { serveStatic } from './serve-static'
 
-const module = <E extends Env = Env>(
-  options: Omit<ServeStaticOptions<E>, 'namespace'> = { root: '' }
-) => {
-  options.manifest ??= manifest
+const module = <E extends Env = Env>(options: Omit<ServeStaticOptions<E>, 'namespace'>) => {
   return serveStatic<E>(options)
 }
 

--- a/src/adapter/cloudflare-workers/serve-static.test.ts
+++ b/src/adapter/cloudflare-workers/serve-static.test.ts
@@ -32,13 +32,14 @@ Object.assign(global, {
 describe('ServeStatic Middleware', () => {
   const app = new Hono()
   const onNotFound = vi.fn(() => {})
-  app.use('/static/*', serveStatic({ root: './assets', onNotFound }))
-  app.use('/static-no-root/*', serveStatic())
+  app.use('/static/*', serveStatic({ root: './assets', onNotFound, manifest }))
+  app.use('/static-no-root/*', serveStatic({ manifest }))
   app.use(
     '/dot-static/*',
     serveStatic({
       root: './assets',
       rewriteRequestPath: (path) => path.replace(/^\/dot-static/, '/.static'),
+      manifest,
     })
   )
 
@@ -106,8 +107,8 @@ describe('With options', () => {
 
 describe('With `file` options', () => {
   const app = new Hono()
-  app.get('/foo/*', serveStatic({ path: './assets/static/hono.html' }))
-  app.get('/bar/*', serveStatic({ path: './static/hono.html', root: './assets' }))
+  app.get('/foo/*', serveStatic({ path: './assets/static/hono.html', manifest }))
+  app.get('/bar/*', serveStatic({ path: './static/hono.html', root: './assets', manifest }))
 
   it('Should return hono.html', async () => {
     const res = await app.request('http://localhost/foo/fallback')
@@ -135,7 +136,7 @@ describe('With middleware', () => {
 
   app.use('/static/*', md1)
   app.use('/static/*', md2)
-  app.use('/static/*', serveStatic({ root: './assets' }))
+  app.use('/static/*', serveStatic({ root: './assets', manifest }))
   app.get('/static/foo', (c) => {
     return c.text('bar')
   })
@@ -171,6 +172,7 @@ describe('Types of middleware', () => {
         onNotFound: (_, c) => {
           expectTypeOf(c.env).toEqualTypeOf<Env['Bindings']>()
         },
+        manifest,
       })
     )
   })

--- a/src/adapter/cloudflare-workers/serve-static.ts
+++ b/src/adapter/cloudflare-workers/serve-static.ts
@@ -9,7 +9,7 @@ import { getContentFromKVAsset } from './utils'
 export type ServeStaticOptions<E extends Env = Env> = {
   root?: string
   path?: string
-  manifest?: object | string
+  manifest: object | string
   namespace?: KVNamespace
   rewriteRequestPath?: (path: string) => string
   onNotFound?: (path: string, c: Context<E>) => void | Promise<void>
@@ -19,7 +19,7 @@ const DEFAULT_DOCUMENT = 'index.html'
 
 // This middleware is available only on Cloudflare Workers.
 export const serveStatic = <E extends Env = Env>(
-  options: ServeStaticOptions<E> = { root: '' }
+  options: ServeStaticOptions<E>
 ): MiddlewareHandler => {
   return async (c, next) => {
     // Do nothing if Response is already set

